### PR TITLE
[stable/openvpn] Kubernetes 1.16 compatibility

### PR DESCRIPTION
**Pull request by srueg**
#### What this PR does / why we need it:
Update api versions (deprecated -> actual) to support k8s 1.16
ref: https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/openvpn/templates/openvpn-deployment.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-apiVersion: apps/v1beta2
+apiVersion: apps/v1
```
Comment (jasongwartz) [inline stable/openvpn/templates/openvpn-deployment.yaml:1]: This is the only substantive change here, right?

Comment (srueg) [reply stable/openvpn/templates/openvpn-deployment.yaml:1]: Yes, correct

**Comment by jasongwartz** [comment]
Thanks for the contribution!

**Pull request merged**